docs/configuration/general/styling.rst

```diff
@@ -97,6 +97,23 @@ The configuration in the admin requires this to be provided as JSON, for example
 
 .. _style dictionary: https://amzn.github.io/style-dictionary/
 
+**Style of required fields in forms**
```

Comment:
I don't think this is the best place for this documentation, https://open-forms.readthedocs.io/en/latest/manual/forms/index.html is probably a better candidate

Reply:
Since this other section is in Dutch, I translated it. I am not sure about the translation for the name of the new config model fields though... they are not super good names in English either :thinking:

Reply:
thanks, we'll take a look at the translations before merging :)